scripts/generate_embeddings.py

```diff
@@ -88,15 +89,23 @@ def got_whitespace(text: str) -> bool:
         help="HF repo id of the embedding model",
     )
     parser.add_argument(
-        "-c", "--chunk", type=int, default="1500", help="Chunk size for embedding"
+        "-c", "--chunk", type=int, default=380, help="Chunk size for embedding"
     )
     parser.add_argument(
-        "-l", "--overlap", type=int, default="10", help="Chunk overlap for embedding"
+        "-l", "--overlap", type=int, default=0, help="Chunk overlap for embedding"
+    )
+    parser.add_argument(
+        "-em",
+        "--exclude-metadata",
+        nargs="+",
+        default=None,
+        help="Metadata to be excluded during embedding",
```

Comment:
if we're not excluding any metadata, are we addressing https://issues.redhat.com/browse/OLS-609 with this PR?